server/graphql/invoice/src/mutations/customer_return/insert.rs

```diff
@@ -131,6 +131,7 @@ impl InsertInput {
                 .into_iter()
                 .map(|line| line.to_domain())
                 .collect(),
+            is_patient_return: false,
```

Comment:
Needed to distinguish the usual returns, from a customer store, from the new patient returns..